apps/examples/testcase/le_tc/kernel/itc_libc_spawn.c

```diff
@@ -0,0 +1,495 @@
+
+	ret_chk = posix_spawnattr_getschedpolicy(&st_attr, &schedpolicy);
+	TC_ASSERT_EQ("posix_spawnattr_getschedpolicy", ret_chk, ERROR);
+	TC_ASSERT_EQ("posix_spawnattr_getschedpolicy", get_errno(), EINVAL);
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @fn                   : itc_libc_spawn_posix_spawnattr_setgetsigmask_n
+* @brief                : sets and gets the spawn-sigmask attribute in an initialized attributes object referenced by st_attr
+* @scenario             : posix_spawnattr_setsigmask with invalid params in setter as well as in getter.
+* @API's covered        : posix_spawnattr_setsigmask, posix_spawnattr_getsigmask
+* @Preconditions        : none
+* @Postconditions       : none
+* @Return               : void
+*/
+static void itc_libc_spawn_posix_spawnattr_setgetsigmask_n(void)
+{
+	posix_spawnattr_t st_attr;
+	int ret_chk = ERROR;
+	sigset_t setMask = MIN_SIGNO;
+	sigset_t getMask;
+
+	ret_chk = posix_spawnattr_setsigmask(NULL, &setMask);
+	TC_ASSERT_EQ("posix_spawnattr_setsigmask", ret_chk, ERROR);
+	TC_ASSERT_EQ("posix_spawnattr_setsigmask", get_errno(), EINVAL);
+
+	ret_chk = posix_spawnattr_getsigmask(&st_attr, &getMask);
+	TC_ASSERT_EQ("posix_spawnattr_getsigmask", ret_chk, ERROR);
+	TC_ASSERT_EQ("posix_spawnattr_getsigmask", get_errno(), EINVAL);
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @fn                   :itc_libc_spawn_task_spawnattr_setgetstacksize_n
+* @brief                :sets and gets the stack size attribute of the thread attributes object referred to by st_attr
+* @scenario             :task_spawnattr_setstacksize with invalid params in setter as well as in getter.
+* @API's covered        :posix_spawnattr_init, task_spawnattr_setstacksize,task_spawnattr_getstacksize
+* @Preconditions        :posix_spawnattr_init
+* @Postconditions       :none
+* @Return               :void
+*/
+static void itc_libc_spawn_task_spawnattr_setgetstacksize_n(void)
+{
+	int ret_chk = ERROR;
+	posix_spawnattr_t st_attr;
+	size_t stacksize;
+
+	ret_chk = task_spawnattr_setstacksize(NULL, ITC_SPAWN_STACKSIZE);
+	TC_ASSERT_EQ("task_spawnattr_setstacksize", ret_chk, ERROR);
+	TC_ASSERT_EQ("task_spawnattr_setstacksize", get_errno(), EINVAL);
+
+	ret_chk = task_spawnattr_getstacksize(&st_attr, &stacksize);
+	TC_ASSERT_EQ("task_spawnattr_getstacksize", ret_chk, ERROR);
+	TC_ASSERT_EQ("task_spawnattr_getstacksize", get_errno(), EINVAL);
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @fn                   : itc_libc_spawn_posix_spawn_file_actions_init_n
+* @brief                : Initializes the object referenced by file_action
+* @scenario             : initializes the object referenced by st_fileactions, to an empty set of file actions for subsequent use
+* @API's covered        : posix_spawn_file_actions_init, posix_spawn_file_actions_destroy
+* @Preconditions        : none
+* @Postconditions       : posix_spawn_file_actions_destroy
+* @Return               : void
+*/
+static void itc_libc_spawn_posix_spawn_file_actions_init_n(void)
+{
```

Comment:
This ITC is not being called from main function. Also it does not seem to be ITC, please remove it